src/app/pages/apps/components/docker-registries/docker-registry-form/docker-registry-form.component.html

```diff
@@ -0,0 +1,67 @@
+<ix-modal-header
+  [requiredRoles]="requiredRoles"
+  [title]="title | translate"
+  [loading]="isFormLoading"
+></ix-modal-header>
+
+<mat-card>
+  <mat-card-content>
+    <form class="ix-form-container" [formGroup]="form" (submit)="onSubmit()">
+      <ix-fieldset>
+        @if (!isLoggedInToDockerHub && !existingDockerRegistry) {
+          <ix-select
+            formControlName="registry"
+            [label]="'URI' | translate"
+            [options]="registriesOptions$"
+            [hideEmpty]="true"
+          ></ix-select>
+        }
+
+        @if (existingDockerRegistry || isLoggedInToDockerHub || !form.controls.registry.value) {
+          <ix-input
+            formControlName="uri"
+            [label]="'URI' | translate"
+            [required]="true"
+          ></ix-input>
+        }
+
+        <ix-input
+          formControlName="name"
```

Comment:
Let's hide the name field and default it to Docker Hub when Docker Hub is selected.